hardware/raspberrypi/bootmodes/pi4_bootloader_config.md

```diff
@@ -1,6 +1,11 @@
 # Pi4 Bootloader Configuration (BETA)
```

Comment:
This isn't beta

Reply:
Ah, I see what's happened here. All of this stuff is beta, we may as well just leave this on the beta release notes in Github and merge them into boot_eeprom.md when the beta is released.